cfme/scripting/ipyshell.py

```diff
@@ -14,9 +13,12 @@
 
 
 @click.command(help="Open an IPython shell")
-def main():
+@click.option('--no-quickstart', is_flag=True)
+def main(no_quickstart):
     """Use quickstart to ensure we have correct env, then execute imports in ipython and done."""
-    quickstart.main(quickstart.parser.parse_args(['--mk-virtualenv', sys.prefix]))
+    if not no_quickstart:
```

Comment:
**required** i just noted that this quick-start usage ins incorrect as loaded packages dont get updates, this can lead to inconsistent system state - a solution similar to runtest may be needed (which also starts a new process)